homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -198,10 +226,10 @@ async def fetch_data(self):
                 _LOGGER.error(
```

Comment:
We shouldn't log an error when raising `UpdateFailed`. That would spam the log. Just include the error message in the exception argument.